spec/metrician_spec.rb

```diff
@@ -271,6 +271,28 @@ def app
       end
     end
 
+    describe "middleware exceptions" do
+      def app
+        require "metrician/middleware/request_timing"
+        require "metrician/middleware/application_timing"
+        Rack::Builder.app do
+          use Metrician::Middleware::RequestTiming
+          use Metrician::Middleware::ApplicationTiming
+          run lambda { |env| raise "boom" }
+        end
+      end
+
+      specify "middleware exceptions don't cause errors in response size tracking" do
+        Metrician.configuration[:request_timing][:response_size][:enabled] = true
```

Comment:
I decided to do this and took this approach: https://github.com/Instrumental/metrician-ruby/commit/6a69af92af287af5ef377c0dfb29b601bbd6c66a 

Feel free to yank that if you think it is a bad one.